rice.gemspec

```diff
@@ -100,20 +66,16 @@ of the Ruby C API.
 
     # Test source files
     'test/*.?pp',
-    'test/ext/Makefile.am',
-    'test/ext/Makefile.in',
+    'test/extconf.rb',
     'test/ext/t1/extconf.rb',
     'test/ext/t1/*.*pp',
     'test/ext/t2/extconf.rb',
     'test/ext/t2/*.*pp'
   ]
 
-  s.files = patterns.collect { |p| Dir.glob(p) }.flatten
-
-  s.required_ruby_version = ">= 2.4"
+  s.required_ruby_version = ">= 2.7"
```

Comment:
Can we bump this to `2.5`? Trying to support all versions that the Ruby team supports.

Reply:
Sure. Note that Ruby 2.5 will no longer be supported as on March 31 though.